server/location/externals.js

```diff
@@ -0,0 +1,44 @@
+const postcodes = require('node-postcodes.io');
```

Comment:
doesn't seem like this file is being used, so it should be moved to a 'test' directory or removed